Resources/public/css/styles.css

```diff
@@ -345,6 +345,28 @@ td.sonata-ba-list-field .editable-empty:not(.editable-open) {
     width: auto; /* See https://github.com/sonata-project/SonataAdminBundle/issues/2950 */
 }
 
+/* icheck */
+.checkbox div[class^="icheckbox"],
```

Comment:
This is a compatibility problem between bootstrap 3 and iCheck.

Bootstrap 3 add a left padding on .checkbox > label and then a negative left margin on the absolute input.

```css
.checkbox label, .radio label {
    min-height: 20px;
    padding-left: 20px;
    margin-bottom: 0;
    font-weight: 400;
    cursor: pointer;
}
.checkbox input[type=checkbox], .checkbox-inline input[type=checkbox], .radio input[type=radio], .radio-inline input[type=radio] {
    position: absolute;
    margin-top: 4px\9;
    margin-left: -20px;
}
```

I checked AdminLTE examples and they don't wrap the checkbox with `div.checkbox`.

I can change the condition to wrap with `div.checkbox` only if the `sonata_admin.options['use_icheck']` is false.